server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -4,7 +4,7 @@
 version: '3.8'
 services:
   cms:
-    image: taccwma/protx-cms:latest
+    image: taccwma/protx-cms:5a09f44
```

Comment:
👍

Reply:
Awesome! I knew there was something different about the count value I was missing. Committed your suggested changes. G2G!